src/Voron/Impl/Paging/CryptoPager.cs

```diff
@@ -83,6 +90,19 @@ public EncryptionBuffer(EncryptionBuffersPool pool)
         public NativeMemory.ThreadStats AllocatingThread;
         public long Generation;
         public bool SkipOnTxCommit;
+        private int _usages;
+
+        public bool CanRelease => _usages == 1;
```

Comment:
Why `1`? Before we check this we call `buffer.ReleaseRef()` so why we don't get `0`?

Reply:
changed